README.md

```diff
@@ -31,7 +31,22 @@ This will expose the following Solr endpoints:
 * http://localhost:8983/solr/ckan-2.9
 * http://localhost:8983/solr/master
 
-### Additional info
+### Building the images
+
+For Solr 6 based images (ie CKAN 2.6 to 2.9), go to the `solr-6` directory and use the Makefile included:
+
+    # Default version in 2.9
+    make build  
+    make build CKAN_VERSION=2.8 
+    make build-multi
```

Comment:
Shouldn't we edit the Makefile so the build command don't create a `master` tag with solr 6? Are we planning on supporting 6 and 8 on master?

Also, I would limit the CKAN_VERSIONS in the Makefile to `2.8 2.9`. (Do we want to keep supporting `2.6 2.7`?)